tests/test_token.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+import nb2workflow
+import logging
+import threading
+import time
+import os
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture
+def app():
+    testfiles_path = os.path.join(os.path.dirname(__file__), 'testfiles')
+    app = nb2workflow.service.app
+    app.notebook_adapters = nb2workflow.nbadapter.find_notebooks(testfiles_path)
+    nb2workflow.service.setup_routes(app)
+    print("creating app")
+    return app
+
+def test_token(client):
+    status_callback_file = "status.json"
+    callback_url = 'file://' + status_callback_file
+    token = 'abc123'
+    query_string = dict(
+        a=20,
+        _async_request='yes',
```

Comment:
Better test both sync and async requests.

Reply:
Done